flocker/cli/script.py

```diff
@@ -28,6 +28,13 @@
 from ._sshconfig import DEFAULT_SSH_DIRECTORY, OpenSSHConfiguration
 
 
+FEEDBACK_CLI_TEXT = (
+    "\n"
+    "If you have any issues or feedback, you can talk to us: "
+    "http://docs.clusterhq.com/en/latest/gettinginvolved/"
```

Comment:
I also suggest adding an extra line break at the end to separate synopsis from description.